### PR TITLE
add card component to storybook

### LINKED ISSUE #18: add sound player to components

### PR DESCRIPTION
**Pull request by mewiemers**
- add types for the animal card to type.ts
- add the full info card for animals to storybook

### CODE REVIEW

File: components/animalcard/Animalcard.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+import { Animal } from "../../utils/type";
+
+export type AnimalProps = {
+  animals: Animal;
+};
+
+function AnimalCard({ animals }: AnimalProps) {
+  return (
+    <div>
+      <h1></h1>
```
Comment (mariegross) [inline components/animalcard/Animalcard.tsx]:
```suggestion
      <h1>{animals.name}</h1>
```

File: components/animalcard/Animalcard.tsx [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+      <div>
+        <img src={animals.ImgSrc} />
+        <p></p>
```
Comment (mariegross) [inline components/animalcard/Animalcard.tsx]:
```suggestion
        <p>{animals.description}</p>
```

**Review by mariegross — APPROVED** [review]
lgtm 🥳

**Review by DNaunin — APPROVED** [review]
Looks great! 🐧

File: components/animalcard/Animalcard.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+      <h1 className={styles.headline}>{name}</h1>
+      <img src={ImgSrc} />
+      <p className={styles.textbox}>{description}</p>
+      <audio controls src={audioFile}>
+        Your browser does not support the
+        <code>audio</code> element.
+      </audio>
+    </div>
+  );
+}
```
Comment (phlgr) [inline components/animalcard/Animalcard.tsx:16]: 👍

**Review by phlgr — APPROVED** [review]
GJ 🚀

**Review by sweisz — COMMENTED** [review]
GJ 👍

**Review by mrana17 — APPROVED** [review]
Destructuring Master 👩‍🎓

**Comment by HorstFrank** [comment]
🚀🚀 Liftoff 🚀🚀

**Pull request merged**